apps/agent-service/src/interface/agent-service.interface.ts

```diff
@@ -183,6 +184,15 @@ export interface IStoreOrgAgentDetails {
   agentType?: string;
 }
 
+export interface StoreDidDetails {
```

Comment:
We are following the prefix 'I' before the interface name

Reply:
```suggestion
export interface IStoreDidDetails {
```